ush/config_defaults.yaml

```diff
@@ -3052,4 +3052,6 @@ cpl_aqm_parm:
   DCOMINpt_src: "/path/to/point/source/base/directory"
 
   DCOMINairnow: "/path/to/airnow/obaservation/data"
-  AQM_AIRNOW_HIST_DIR: "/path/to/historical/airnow/data/dir"
+    #AQM_AIRNOW_HIST_DIR: "/path/to/historical/airnow/data/dir"
```

Comment:
Remove this line if not necessary

Reply:
Will have a global developer code clean up later.